app/controllers/expenses_additional_controller.rb

```diff
@@ -79,11 +73,9 @@ def application_attributes
       monthly_care_expenses: app.monthly_care_expenses,
       monthly_medical_expenses: app.monthly_medical_expenses,
       monthly_court_ordered_expenses: app.monthly_court_ordered_expenses,
-      monthly_tax_deductible_expenses: app.monthly_tax_deductible_expenses,
       care_expenses: app.care_expenses,
       medical_expenses: app.medical_expenses,
       court_ordered_expenses: app.court_ordered_expenses,
-      tax_deductible_expenses: app.tax_deductible_expenses,
     }
```

Comment:
Realized when taking a second look per your feedback that this is unused code so deleted at 86d3ab4 📉  -- good eye!